views/tables/AdminUsersTable/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import Settings from '@mui/icons-material/Settings'
+import Clear from '@mui/icons-material/Clear'
+import LockOpen from '@mui/icons-material/LockOpen'
+import IconButton from '@mui/material/IconButton'
+import Checkbox from '@mui/material/Checkbox'
+import { DatePicker } from '@mui/x-date-pickers/DatePicker'
+import moment from 'moment'
+
+import Table from '../Table'
+
+const AdminUsersTable = (props) => {
+  const headers = [
+    {
+      dataProperty: 'uid',
+      label: 'Username',
+      sortable: true,
```

Comment:
```suggestion
      sortable: true,
```
Was the table sortable in past iterations..?

Reply:
Oh, I'm not sure. It doesn't work so I'm guessing no. I'll remove

Reply:
b408355202da0b77d1904a32e4c87ea2a5f99e29